src/test/java/org/sagebionetworks/bridge/workers/dynamodb/streams/StreamRecordProcessorTest.java

```diff
@@ -0,0 +1,67 @@
+package org.sagebionetworks.bridge.workers.dynamodb.streams;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.slf4j.Logger;
+
+import com.amazonaws.services.dynamodbv2.model.Record;
+import com.amazonaws.services.dynamodbv2.model.StreamRecord;
+import com.amazonaws.services.dynamodbv2.streamsadapter.model.RecordAdapter;
+import com.amazonaws.services.kinesis.clientlibrary.interfaces.IRecordProcessorCheckpointer;
+
+public class StreamRecordProcessorTest {
+
+    private final StreamRecordProcessor processor;
+
+    public StreamRecordProcessorTest() {
+        processor = new StreamRecordProcessor(2) {
+            @Override
+            Logger log() {
+                return mock(Logger.class);
```

Comment:
Would it be better to use a real logger? That way, if unit tests fail, we can use the logs to help figure out what happened.